10_temperature_object/temperature.rb

```diff
@@ -1,39 +1,40 @@
 # frozen_string_literal: true
 
 class Temperature
-  def initialize(hash = {})
-    @hash = hash
+  def initialize(hash)
+    @ftemp = hash[:f]
+    @ctemp = hash[:c]
+    @ftemp = (@ctemp.to_f * 9 / 5) + 32 unless @ctemp.nil?
+    @ctemp = (@ftemp - 32) * 5 / 9 unless @ftemp.nil?
```

Comment:
Avoid computation in the constructor, it may be not needed, and it's just waste of CPU cycles.


Try to apply the strategy pattern.